server/src/schemas/usersSchema.ts

```diff
@@ -0,0 +1,73 @@
+import { gql } from "apollo-server-express";
+
+export const UsersTypeDefs = gql`
+  type User {
+    id: ID!
+    first_name: String!
+    last_name: String!
+    email: String!
+    type: Type!
+    privilege: Privilege!
+    registration_date: Date!
+    balance: Int!
+    hold: Boolean!
+    holds: [Hold]
+    training_modules: [TrainingModule]
+    year: Int
+    college: String
+    major: String
+    description: String
+  }
+
+  enum Type{
+    STUDENT
+    FACULTY
+    EMPLOYEE
+  }
+
+  enum Privilege{
+    MAKER
+    LABBIE
+    ADMIN
+  }
+
+  input StudentUserInput {
+    first_name: String!
+    last_name: String!
+    email: String!
+    privilege: Privilege!
+    balance: Int!
+    hold: Boolean!
+    year: Int!
+    college: String!
+    major: String!
+  }
+
+  input FacultyUserInput {
+    first_name: String!
+    last_name: String!
+    email: String!
+    type: Type!
+    privilege: Privilege!
+    balance: Int!
+    hold: Boolean!
+  }
+
+  type Query {
+    user: User
+    users: [User]
```

Comment:
I'm not a GraphQL expert but I'm pretty sure we don't need `users` here, just `user`.

@smellslikekeenspirit @pjw9963 - can you guys weigh in?

Reply:
Sounds right after doing more reading ngl

Reply:
If you can get list of users using just the `user` query then we don't need the `users` query, which I believe is how it works